test/doctool/test-doctool-html.js

```diff
@@ -5,6 +5,8 @@ const assert = require('assert');
 const fs = require('fs');
 const path = require('path');
 
+common.globalCheck = false;
```

Comment:
This should best come with a comment stating the names of the additional globals